examples/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "1.0.0",
   "private": "true",
   "dependencies": {
-    "@vscode/codicons": "^0.0.33",
+    "@vscode/codicons": "~0.0.35",
     "express": "^4.18.2",
-    "inversify": "^6.0.1",
+    "inversify": "6.0.1",
```

Comment:
I'm not sure about this change. Why not allow compatible versions? There's also 6.0.2 now.

Reply:
From the description: "The inversify version has been fixed to 6.0.1 as 6.0.2 introduces breaking changes. This became required after re-generation yarn.lock"

Reply:
I will revert the `yarn.lock` regeneration.